kotsadm/pkg/handlers/kurl_get.go

```diff
@@ -1 +1,42 @@
 package handlers
+
+import (
+	"net/http"
+
+	"github.com/replicatedhq/kots/kotsadm/pkg/k8s"
+	"github.com/replicatedhq/kots/kotsadm/pkg/kurl"
+	"github.com/replicatedhq/kots/kotsadm/pkg/logger"
+)
+
+func GetKurlNodes(w http.ResponseWriter, r *http.Request) {
+	if handleOptionsRequest(w, r) {
+		return
+	}
+
+	if err := requireValidSession(w, r); err != nil {
+		logger.Error(err)
+		return
+	}
+
+	client, err := k8s.Clientset()
+	if err != nil {
+		logger.Error(err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	mc, err := k8s.Metricsset()
```

Comment:
I think this should be inline, none of the other specialized clientsets are wrapped in that k8s helper, and this is the only use of that function